openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx

```diff
@@ -564,11 +563,11 @@ const DatabaseSchemaPage: FunctionComponent = () => {
     },
   ];
 
-  if (isLoading) {
+  if (isPermissionsLoading) {
     return <Loader />;
   }
 
-  if (!viewDatabaseSchemaPermission) {
+  if (!isPermissionsLoading && !viewDatabaseSchemaPermission) {
```

Comment:
Isn't that obvious that `isPermissionsLoading` will always be false at this point?